System.cmake

```diff
@@ -1,16 +1,20 @@
 # sets ${SYSTEM}
```

Comment:
Improve documentation?
Output:
* SYSTEM, a human-readable string identifier for the current platform (Win32, Darwin or Linux)
* OSX_VERSION, the two numbers OS X version (e.g. 10.9)